docs/source/intro-clx-periodicity-detection.ipynb

```diff
@@ -0,0 +1,138 @@
+{
```

Comment:
<p>is it possible to show and describe the output here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/378/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Added output description. Thank you!</p><p> </p>